plugins/LilyPad/LilyPad.cpp

```diff
@@ -1224,6 +1228,23 @@ u8 CALLBACK PADpoll(u8 value)
                     Update(query.port, query.slot);
                     ButtonSum *sum = &pad->sum;
 
+                    if (config.padConfigs[query.port][query.slot].type == MousePad) {
+                        u8 b1 = 0xFC;
+                        if (sum->buttons[5 + 4] > 0) // Left button
+                            b1 -= 2 << 2;
+                        if (sum->buttons[6 + 4] > 0) // Right button
+                            b1 -= 2 << 1;
```

Comment:
Actually, is there any meaning to 5 + 4, 6 + 4?

Side note: I suppose I also find `2 << 2`, `2 << 1` a bit strange - I'd be a lot more likely to write `1 << 3`, `1 << 2` or simply 8 and 4.